packages/devtools/lib/src/main.dart

```diff
@@ -93,6 +93,11 @@ class PerfToolFramework extends Framework {
         toast('Device connection lost.');
       }
     });
+
+    // Listen for clicks on the 'send feedback' button.
+    queryId('send-feedback-button').onClick.listen((_) {
```

Comment:
add a TODO to fill in the flutter version and the version of this tool that the user was using.